spec/features/child_benefit_tax_calculator_spec.rb

```diff
@@ -7,6 +7,14 @@
     stub_request(:get, Plek.new.find("content-store") + "/content/child-benefit-tax-calculator/main").to_return(body: {}.to_json)
   end
 
+  before do
+    Timecop.travel("2020-04-02")
+  end
+
+  after do
+    Timecop.return
+  end
+
```

Comment:
```suggestion
  around do |example|
    Timecop.travel("2020-04-02") { example.run }
  end
```

This might be slightly neater.